backend/src/litigation_api/ingestion/client.py

```
@@ -0,0 +1,109 @@
+        wait=wait_exponential(multiplier=1, min=2, max=10),
+        retry=retry_if_exception_type((httpx.RequestError, httpx.HTTPStatusError, CRLCAClientError)),
+        reraise=True
+    )
+    async def _fetch_url(self, url: str) -> Dict[str, Any]:
+        """Fetch a specific URL with retry logic."""
+        try:
+            response = await self.client.get(url)
+            response.raise_for_status()
+            return response.json()
+        except httpx.HTTPStatusError as e:
+            logger.error(f"HTTP error fetching {url}: {e.response.status_code}")
```

Ruff (0.15.11)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)